cmd/juju/space/show_test.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2015 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package space_test
+
+import (
+	"github.com/juju/errors"
+	jc "github.com/juju/testing/checkers"
+	gc "gopkg.in/check.v1"
+
+	"github.com/juju/juju/apiserver/params"
+	"github.com/juju/juju/cmd/juju/space"
+)
+
+type ShowSuite struct {
+	BaseSpaceSuite
+}
+
+var _ = gc.Suite(&ShowSuite{})
+
+func (s *ShowSuite) SetUpTest(c *gc.C) {
+	s.BaseSpaceSuite.SetUpTest(c)
+	s.newCommand = space.NewShowSpaceCommand
+}
+
+func (s *ShowSuite) TestRunShowSpaceSucceeds(c *gc.C) {
+	expectedStdout := `space:
+  id: "0"
+  name: alpha
+  subnets: []
```

Comment:
Can we test the output for a space with subnets here?

I got this example:
```yaml
space:
  id: "3"
  name: telemetry
  subnets:
  - cidr: 10.118.99.0/24
    providerid: "18"
    providernetworkid: ""
    providerspaceid: ""
    vlantag: 0
    life: alive
    spacetag: space-telemetry
    zones: []
    status: in-use
  error: null
applications:
- postgresql
machinecount: 0
error: null
```

I think we should omit some of the info if empty, to cut down on noise. Particularly the provider IDs. I think the multi-word keys should be hyphenated.